apps/codeforafrica/src/pages/api/admin/config.yml.page.js

```diff
@@ -0,0 +1,29 @@
+import yaml from "js-yaml";
+
+import config from "./config.json";
+
+import site from "@/codeforafrica/utils/site";
+
+config.logoUrl = site.logoUrl;
+
+export default function handler(req, res) {
+  if (req.method === "GET") {
+    if (process.env.NODE_ENV === "production") {
+      // Set production configurations
+      config.backend.name = "github";
+      config.backend.repo = process.env.GITHUB_BACKEND_REPO;
+      config.backend.base_url = site.url.replace(/\/+$/, "");
+      config.backend.auth_endpoint = process.env.GITHUB_AUTH_ENDPOINT;
+      config.publish_mode = "editorial_workflow";
+      // Remove dev configurations
+      config.local_backend = undefined;
+    }
```

Comment:
Any reason why we're not setting `logoUrl` inside the `if` check?

```suggestion
export default function handler(req, res) {
  if (req.method === "GET") {
    if (process.env.NODE_ENV === "production") {
      // Set production configurations
      config.backend.name = "github";
      config.backend.repo = process.env.GITHUB_BACKEND_REPO;
      config.backend.base_url = site.url.replace(/\/+$/, "");
      config.backend.auth_endpoint = process.env.GITHUB_AUTH_ENDPOINT;
      config.publish_mode = "editorial_workflow";
      // Remove dev configurations
      config.local_backend = undefined;
    }
    config.logoUrl = site.logoUrl;
```

Reply:
not really